src/components/common/toast/toast.styles.tsx

```diff
@@ -0,0 +1,6 @@
+import { Color, Text, Toast } from "@lifesg/react-design-system";
+import styled from "styled-components";
+
+export const ToastWrapper = styled(Toast)`
+    width: fit-content;
```

Comment:
what is it meant to address? the component's width is already scaling according to its content